pkg/gen/input/workflows/internal/file/check_values_schema.yaml.template

```diff
@@ -26,6 +27,13 @@ jobs:
 
       - name: 'Check if values.yaml is a valid instance of values.schema.json'
         run: |
-          HELM_DIR=$(dirname $(git diff --name-only origin/${GITHUB_BASE_REF} origin/${GITHUB_HEAD_REF} \
-           | grep 'helm/[-a-z]*\/values\.' | head -1))
-          ${HOME}/yajsv -s ${HELM_DIR}/values.schema.json ${HELM_DIR}/values.yaml
+          HELM_DIR=$(git diff --name-only origin/${GITHUB_BASE_REF} origin/${GITHUB_HEAD_REF} \
+           | grep 'helm/[-a-z].*\/' | head -1 | awk -F '/' '{print $1"/"$2}')
+          VALUES=${HELM_DIR}/values.yaml
+          if [ -f ${HELM_DIR}/ci/ci-values.yaml ]; then
+            # merge ci-values.yaml into values.yaml (providing required values)
+            yq '. *= load("'${HELM_DIR}'/ci/ci-values.yaml")' ${HELM_DIR}/values.yaml | tee ${HELM_DIR}/combined-values.yaml
```

Comment:
[nit]

Is `tee` on purpose to print the merged result in CI? Then maybe surround with `printf "Merged values:\n=====; yq ...; echo "====="`.

Reply:
amended